### PR TITLE
Create kth_smallest_element_algorithm.c

### PR DESCRIPTION
**Pull request by Vaibhav1671998**
Added code to find kth smallest element in c Language

### CODE REVIEW

**Comment by Vaibhav1671998** [comment]
Have a look at this @arnavb

File: code/randomized_algorithms/kth_smallest_element_algorithm/kth_smallest_element_algorithm.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        }
+    }
+    return a[k-1];
+}
+int main()
+{
+    int n;
+    printf("Enter number of elements : ");
+    scanf("%d",&n);
+    int a[n];
```
Comment (arnavb) [inline code/randomized_algorithms/kth_smallest_element_algorithm/kth_smallest_element_algorithm.c]: Better name

File: code/randomized_algorithms/kth_smallest_element_algorithm/kth_smallest_element_algorithm.c [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    printf("Enter Elements : \n");
+    for (int i = 0; i < n; i++)
```
Comment (arnavb) [inline code/randomized_algorithms/kth_smallest_element_algorithm/kth_smallest_element_algorithm.c]: Elements -> elements

File: code/randomized_algorithms/kth_smallest_element_algorithm/kth_smallest_element_algorithm.c [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        scanf("%d",&a[i]);
+    }
+    int k;
```
Comment (arnavb) [inline code/randomized_algorithms/kth_smallest_element_algorithm/kth_smallest_element_algorithm.c]: Same as above.

File: code/randomized_algorithms/kth_smallest_element_algorithm/kth_smallest_element_algorithm.c [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    printf("Enter k (nth smallest element to find) : ");
+    scanf("%d",&k);
+    int result = find_element(a,n,k);
```
Comment (arnavb) [inline code/randomized_algorithms/kth_smallest_element_algorithm/kth_smallest_element_algorithm.c]: No need for the variable.

**Comment by Vaibhav1671998** [comment]
I had made the changes @arnavb .

I don't think default initialization is needed as i am taking the input 
Names have been changed
Extra variable is removed .

**Comment by arnavb** [comment]
Enter Elements should be Enter elements

**Comment by Vaibhav1671998** [comment]
Merge this plzz @AdiChat

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**